azure-pipelines/templates/steps/automation-cert.yml

```diff
@@ -10,18 +10,18 @@ steps:
   inputs:
     filename: echo
     arguments: '##vso[task.setvariable variable=${{ parameters.envVstsMvnAt }}]$(System.AccessToken)'
-- task: AzureKeyVault@1
+- task: AzureKeyVault@2
   displayName: 'Azure Key Vault: Download Cert for Automation'
   inputs:
-    ConnectedServiceName: 9a9308ae-59b4-475f-b49c-3e828c508d8b
-    KeyVaultName: ADALTestInfo
-    SecretsFilter: AndroidAutomationRunner
+    azureSubscription: 'AuthSdkResourceManager'
+    KeyVaultName: 'buildautomation'
+    SecretsFilter: 'AzureADIdentityDivisionTestAgentCert'
```

Comment:
I'm not familiar with this. Is this a new cert? What is this used for?

Reply:
reverted